feature-libs/user/profile/components/reset-password/reset-password.component.ts

```diff
@@ -0,0 +1,22 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { Observable } from 'rxjs';
+import { ResetPasswordService } from './reset-password.service';
+
+@Component({
+  selector: 'cx-reset-password',
+  templateUrl: './reset-password.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ResetPasswordComponent {
+  form: FormGroup = this.service.form;
+  isUpdating$ = this.service.isUpdating$;
+
+  token$: Observable<string> = this.service.resetToken$;
+
+  constructor(protected service: ResetPasswordService) {}
+
+  onSubmit(token: string) {
+    this.service.reset(token);
```

Comment:
I would suggest changing the service's method name to something less generic than just `reset`. 
Seeing this for the first time made me confused (why are we resetting service in onSubmit?).